utilities_common/cli.py

```diff
@@ -51,6 +51,33 @@ def get_command(self, ctx, cmd_name):
 
             ctx.fail('Too many matches: %s' % ', '.join(sorted(matches)))
 
+try:
+    # noinspection PyPep8Naming
+    import ConfigParser as configparser
+except ImportError:
+    # noinspection PyUnresolvedReferences
```

Comment:
This is a PyCharm directive. Suggest removing, as not all developers use PyCharm.